aries-site/src/pages/components/togglegroup.mdx

```diff
@@ -0,0 +1,36 @@
+## Use Cases
+
+### Filtering
+- **Ma**p: Used to provide user with a map view of a datatable or list.
+- **List**: Used to provide user with a list view of a chart or map.
+- **Chart**: Used to provide user with a chart view a datatable or list.
+- **DataTable**: Used to provide user with a DataTable view of a chart or map.
+
+
+## Anatomy
+| Label | Region           | Purpose           |                  Required                   | Notes             |
+| ----- | ---------------- | ----------------- | :-----------------------------------------: | ----------------- |
+| **1** | **Container** | Organizes children, how they appear and spaced. | Required |        --         |
+| **2** | **Label** | Describes what the button will do when selected. | Optional |        --         |
+| **3** | **Icon** | Used to visually represent text label with or without label.  | Optional |       a11yTitle required       |
+
+
+## Types 
+
+Togglegroup can be used in multiple ways:
+
+**Single Option:** Used only when toggling between a single view. For example, toggling between a Chart and DataTable.
+
+**Multiple Options:** Used to select more than one option. For example, selecting mutliple styles like bold and italic within a TextEditor.
+
+
+## Accessibility
+
+In icon-only cases, ensure each icon has a clear a11yTitle.
+
+| Key | Interaction           | 
+| ----- | ---------------- |
+| **Tab** | Focuses on first toggle button. |
```

Comment:
I wonder if we should add focuses on the first togglebutton or the active toggle button

Reply:
Hey Brittany, i'm not sure I understand, can you elaborate more please.

Reply:
yes, so lets say we have `underline` `bold` `italic` in that order and we make the `bold` on as default than the focus would be on the `bold` instead of the first toggle button.